packages/gatsby/gatsby-node.js

```diff
@@ -29,6 +31,26 @@ exports.onCreateWebpackConfig = ({ plugins, getConfig, actions }) => {
     ],
   });
 
+  if (process.env.NODE_ENV === 'production') {
+    actions.setWebpackConfig({
+      plugins: [
+        new SentryWebpackPlugin({
+          include: 'public',
+          ignore: ['app-*', 'polyfill-*', 'framework-*', 'webpack-runtime-*'],
+          // Handle sentry-cli configuration errors when the user has not done it not to break
+          // the build.
+          errorHandler(err, invokeErr) {
+            const { message } = err;
+            if (message.includes('organization slug is required') || message.includes('project slug is required')) {
+              return;
```

Comment:
Does this still log the error if we do an early return?

Reply:
It won't

Reply:
@iker-barriocanal Feel like we need a way to tell users about misconfigured sentry-cli without it being blocking. Can we take advantage of the `debug` flag somehow?

Reply:
What do you mean exactly? Telling users when they haven't correctly configured sentry-cli? If this is the case:

- Enabling `debug` will generate too much noise, and they can always customize this by setting `SENTRY_LOG_LEVEL` env var, see [configuration values](https://docs.sentry.io/product/cli/configuration/#configuration-values).
- The code to check whether the file and its values are correct would be too complex and too hard to maintain for the value it provides. Are we planning to check if a sentry-cli config file exists? Or, checking the format is correct? Or, checking if the values are correct (need to ask Sentry whether the org exists)?

Considering sentry-cli has proper docs, failing doesn't have major consequences (not uploading source maps for a misconfiguration, which they can read the docs of, or try to have their own config, can retry as many times as they wish at no cost (not a production issue)...), and we can update Gatsby docs if the process is clear enough, I don't think it's worth it to provide additional checks. I'd rather remove this UX improvement over adding it and potentially provide a worse UX.

Note that I might have misunderstood what you meant.

Reply:
I just wanted to avoid situations where someone forgets to set an org slug, and doesn't understand why source maps uploading isn't working for gatsby (since the error is being swallowed). 

I think we should add to docs how user's can define their own custom config with the webpack plugin.

Reply:
It's a good point, but I don't think it's worth covering it in this context. As for the docs, we link to the Sentry Webpack docs [here](https://docs.sentry.io/platforms/javascript/guides/gatsby/sourcemaps/uploading/webpack/); although these sections could be improved, is there anything specific you have in mind?